dockerfiles/alpine/mg/Dockerfile

```diff
@@ -83,19 +86,27 @@ RUN \
     && echo '[ ! -z "${TERM}" -a -r /etc/motd ] && printf "$(cat /etc/motd)"' >> ${USER_HOME}"/.bashrc" \
     && apk add --update --no-cache bash libxml2-utils netcat-openbsd \
     && wget --no-check-certificate -O /root/${MGW_SERVER}.zip "${MGW_SERVER_DIST_URL}" \
+    && wget --no-check-certificate -O /tmp/openjdk.tar.gz "${JRE_BINARY_URL}" \
```

Comment:
Can we use the internal jre, directly rather than agin downloading the jre

Reply:
FYI, after an internal discussion we (rajith me and the team) decided to go with external jre so that we have the flexibility to easily upgrade docker image jre without going through an immediate mgw patch release